src/internal/connector/exchange/api/contacts.go

```diff
@@ -52,12 +56,20 @@ func (c Contacts) DeleteContactFolder(
 	return c.stable.Client().UsersById(user).ContactFoldersById(folderID).Delete(ctx, nil)
 }
 
-// RetrieveContactDataForUser is a GraphRetrievalFun that returns all associated fields.
-func (c Contacts) RetrieveContactDataForUser(
+// GetItem retrieves a Contactable item.
+func (c Contacts) GetItem(
 	ctx context.Context,
-	user, m365ID string,
-) (serialization.Parsable, error) {
-	return c.stable.Client().UsersById(user).ContactsById(m365ID).Get(ctx, nil)
+	user, itemID string,
+) (serialization.Parsable, *details.ExchangeInfo, time.Time, error) {
+	cont, err := c.stable.Client().UsersById(user).ContactsById(itemID).Get(ctx, nil)
+	if err != nil {
+		return nil, nil, time.Time{}, err
+	}
+
+	info := ContactInfo(cont)
+	modTime := orNow(cont.GetLastModifiedDateTime())
+
+	return cont, info, modTime, nil
```

Comment:
I'm not really a fan of returning extra values from functions when those values can also be sourced from other returned values (e.x. modTime). It seems like it would be better and less confusing to users of this function to just get the mod time if they need to. This would also remove potential confusion about why the returned mod time doesn't match the mod time in the returned item

If we need to expand an interface to support a `GetLastModifiedDateTime()` call then we can do that

Reply:
I'm with you here.  Mostly kept it in because I wasn't sure why we were handling modtime differently in the first place.  Sounds like it's not wanted, so we can consolidate.

Reply:
>Mostly kept it in because I wasn't sure why we were handling modtime differently in the first place

If you haven't found the answer already, this is because we've made the items implement the StreamModTime interface or something so that we can do kopia-assisted incrementals. In those cases it's safe to use the current time for modtime (in kopia) so that kopia will just re-upload the data

Reply:
Fair.  I figure it's all the same if we default the modified time in the exchangeInfo to `now`, like we were doing with the interface value.  Does that concern you at all?

Reply:
I think we should keep those separate as well. Defaulting it in the item info will affect what the user sees and can hide issues that may crop up